packages/compiler/test/schema/schema_extractor.ts

```diff
@@ -166,7 +182,10 @@ function assertNoMissingTags(descMap: Map<string, string[]>): void {
     extractedTags.push(...key.split('|')[0].split('^')[0].split(','));
   });
 
-  const missingTags = ALL_HTML_TAGS.split(',').filter((tag) => extractedTags.indexOf(tag) == -1);
+  const missingTags = [
+    ...ALL_HTML_TAGS.split(','),
+    ...ALL_MATH_TAGS.split(',').map((tag) => MATH_PREFIX + tag),
+  ].filter((tag) => extractedTags.indexOf(tag) == -1);
```

Comment:
If you feel like changing this

```suggestion
  ].filter((tag) => !extractedTags.includes(tag));
```